pkg/network/nmstate.go

```diff
@@ -32,3 +39,36 @@ func renderNMState(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir string,
 
 	return objs, nil
 }
+
+func cleanUpNMState(conf *opv1alpha1.NetworkAddonsConfigSpec, ctx context.Context, client k8sclient.Client) []error {
+	if conf.NMState == nil {
+		return nil
+	}
+
+	// Get existing
+	existing := &unstructured.Unstructured{}
+	gvk := schema.GroupVersionKind{Group: "apps", Version: "v1", Kind: "Deployment"}
+	existing.SetGroupVersionKind(gvk)
+	namespace := "cluster-network-addons"
```

Comment:
This should be taken from the operand namespace variable.

https://github.com/kubevirt/cluster-network-addons-operator/blob/master/pkg/network/multus.go#L58